README.md

```diff
@@ -82,25 +98,31 @@ Because of current Multi-Targeting limitations with Visual Studio for Mac you wi
 
 Here are a few different options we've put together to help make this process easier
 - Branches 3.5+ come with a Cake script target that you can use to build and open VSMac
-```sh
-./build.sh --target vsmac
-```
-- When working on an earlier branch that does not have the cake scripts then you can use the following [build.sh] script(https://gist.github.com/PureWeen/92c1e1aff0c257c3decf0bcb8d6e9296)
-
-- If you don't want to run any scripts then
+    ```sh
+    ./build.sh --target vsmac
+    ```
+    OR as mentioned above in the Windows section about provisioning, you can also use the new [Cake.Tool](https://cakebuild.net/docs/running-builds/runners/dotnet-tool)
+    ```
+    dotnet tool install Cake.Tool -g
+    dotnet cake --target=provision
+    ```
+    
+- When working on an earlier branch that does not have the cake scripts, you can use the following [build.sh](https://gist.github.com/PureWeen/92c1e1aff0c257c3decf0bcb8d6e9296) script
+
+- If you don't want to run any scripts:
   - Open Xamarin.Forms.sln
   - Wait for VSMAC to finish restoring all projects
   - from the command line run:
     - `msbuild Xamarin.Forms.Build.Tasks/Xamarin.Forms.Build.Tasks.csproj`
   - Now you should be able to run and deploy everything. The only reason you would need to do this process again is if you clean the solution folder or delete the bin/obj folders that are part of the `Xamarin.Forms.Build.Tasks.csproj`
 
-If you are on Visual Studio for Mac 2017 you will need to turn off automatic package restore (Visual Studio => Preferences => Nuget => General => uncheck the Package Restore box) before working on the Xamarin.Forms solution. This step is no longer needed with Visual Studio for Mac 2019
+If you are on Visual Studio for Mac 2017, you will need to turn off automatic package restore (Visual Studio => Preferences => Nuget => General => uncheck the Package Restore box) before working on the Xamarin.Forms solution. This step is no longer needed with Visual Studio for Mac 2019.
```

Comment:
We can delete this part. We've dropped support for 2017 so it's not really relevant anymore

Reply:
sounds good - deleted!